video_xblock/static/html/student_view.html

```diff
@@ -11,3 +12,58 @@ <h3 class="hd hd-2">{{display_name}}</h3>
     height="600px"
 >
 </iframe>
+<ul class="wrapper-downloads-custom">
+    {% comment %}
+                    <!-- Button for download video -->
+        {% if download_video_link %}
```

Comment:
We don't have this feature yet. It's ok to leave a placeholders but no need to add actual code.